velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -80,55 +66,62 @@ TEST_F(ParquetReaderTest, parseSample) {
   auto rowReaderOpts = getReaderOpts(sampleSchema());
   auto scanSpec = makeScanSpec(sampleSchema());
   rowReaderOpts.setScanSpec(scanSpec);
-  auto rowReader = reader.createRowReader(rowReaderOpts);
-  auto expected = vectorMaker_->rowVector(
-      {rangeVector<int64_t>(20, 1), rangeVector<double>(20, 1)});
-  assertReadExpected(sampleSchema(), *rowReader, expected, *pool_);
+  auto rowReader = reader->createRowReader(rowReaderOpts);
+  auto expected = makeRowVector({
```

Comment:
Was there a specific reason to switch from vectorMaker? You could use that also in your test to create the test data same as in the reader. We talked about the different APIs to make vectors (e.g. makeRowVector and makeFlatVector) but it didn't mean that you needed to switch the APIs.
The only other different workload generator is then then batch maker in the sink test which generates multiple batches.

Reply:
I was suggested to use lambda directly to VectorMaker to generate mock data ,so I decided to switch away from this API to make vector. I figured there was code duplication and extra code which wasn't needed, generating data using lambda achieved the same purpose.